lib/cfn-model/transforms/serverless.rb

```diff
@@ -0,0 +1,82 @@
+class CfnModel
+  class Transforms
+    # Handle transformation of model elements performed by the
+    # Serverless trasnform, see
+    # https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/transform-aws-serverless.html
+    class Serverless
+      def perform_transform(cfn_hash)
+        resources = cfn_hash['Resources'].clone
+        resources.each do |r_name, r_value|
+          next unless r_value['Type'].eql? 'AWS::Serverless::Function'
```

Comment:
there are a few beyond the Function: https://awslabs.github.io/serverless-application-model/internals/generated_resources.html

Reply:
perhaps a mapping here between serverless resources and the replace_* method?